pkg/util/k8sutil.go

```diff
@@ -121,3 +123,27 @@ func IsPodFaulty(pod *corev1.Pod) (bool, error) {
 	}
 	return false, nil
 }
+
+func NewEvent(name string, p *v1alpha1.PravegaCluster, reason string, message string, eventType string) *corev1.Event {
+	now := metav1.Now()
+	operatorName, _ := k8s.GetOperatorName()
+	event := corev1.Event{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: p.Namespace,
+		},
+		InvolvedObject: corev1.ObjectReference{
+			Kind:      "pravegacluster",
```

Comment:
This should be:
```
corev1.ObjectReference{
  APIVersion: "pravega.pravega.io/v1alpha1",
  Kind:      "PravegaCluster",
  Name:      p.GetName(),
  Namespace: p.GetNamespace(),
  ResourceVersion: p.GetResourceVersion(),
  UID: p.GetUID(),
}
```


Reference:
https://eos2git.cec.lab.emc.com/NAUT/nautilus-dist/blob/master/charts/nautilus-pravega/templates/nautilus-pravega.yaml#L17